src/mame/drivers/seta.cpp

```diff
@@ -1716,9 +1716,9 @@ ADDRESS_MAP_START(seta_state::downtown_map)
 	AM_RANGE(0x400000, 0x400007) AM_WRITE(twineagl_tilebank_w)      // special tile banking to animate water in twineagl
 	AM_RANGE(0x500000, 0x500001) AM_WRITENOP                        // ?
 	AM_RANGE(0x600000, 0x600003) AM_READ(seta_dsw_r)                // DSW
-	AM_RANGE(0x700000, 0x7003ff) AM_RAM AM_SHARE("paletteram")  // Palette
+	AM_RANGE(0x700000, 0x7003ff) AM_RAM AM_SHARE("paletteram1")  // Palette
 	AM_RANGE(0x800000, 0x800005) AM_WRITEONLY AM_SHARE("vctrl_0")// VRAM Ctrl
-	AM_RANGE(0x900000, 0x903fff) AM_RAM_WRITE(seta_vram_0_w) AM_SHARE("vram_0") // VRAM
+	AM_RANGE(0x900000, 0x903fff) AM_RAM_WRITE(seta_vram_w<0>) AM_SHARE("vram_0") // VRAM
```

Comment:
This will produce compilation errors when seta_vram_w doesn't come before this in the same file.

Reply:
It doesn't have to come before it in the same file - it just has to be in the same file or a header included by it.